plugins/org/src/components/Cards/Group/MembersList/MembersListCard.test.tsx

```diff
@@ -33,9 +33,7 @@ describe('MemberTab Test', () => {
     spec: {
       type: 'team',
       parent: 'boxoffice',
-      ancestors: ['boxoffice', 'acme-corp'],
       children: [],
-      descendants: [],
```

Comment:
Not related to my PR, but somehow slipped through in another one and broke my build 🤔